### PR TITLE
ActiveStorage upgrade command

### LINKED ISSUE #41101: Add 'rails active_storage:update' to upgrade documentations from 6.0 to 6.1
### Steps to reproduce
1. Read [documentation](https://guides.rubyonrails.org/upgrading_ruby_on_rails.html#upgrading-from-rails-6-0-to-rails-6-1) to upgrade from 6.0 to 6.1
2. Run into problems displaying ActiveStorage files when trying to run site after upgrade to 6.1

### Expected behavior
1. Documentation should mention that if you use ActiveStorage, you should run:
```rails active_storage:update```
```rails db:migrate```

### Actual behavior
1. Nothing mentioned about the need to add new columns to the ActiveStorage tables for the upgrade to work.

### System configuration
**Rails version**: 6.0 -> 6.1.1.

### PR DESCRIPTION
**Pull request by michelson**
### Summary

ActiveStorage::Blob is generating error when upgrading from rails 6.0.3.4 to 6.1 without the proper migration
https://github.com/rails/rails/pull/34935#issuecomment-742715565

### CODE REVIEW

**Comment by bogdanvlviv** [comment]
Hi, thanks for raising this issue. Actually, I don't think we should mention this.

If follow basing instructions of how to upgrade rails app to a new version, a user should execute `bin/rails app:update`
(see https://edgeguides.rubyonrails.org/upgrading_ruby_on_rails.html#the-update-task). According to this change(https://github.com/rails/rails/commit/2bad3f46cdd8decee7d2d9b804ac7c75b5a1cc5d#diff-cab1d0cb70e3e5672120311993d2196ba6fdc82244bd697a7bdfa80f6d6768b9R5 it should trigger `update:active_storage` task.

**Comment by georgeclaghorn** [comment]
Agree that we shouldn’t mention `active_storage:update`, but it’s probably still worth noting the multi-service changes and that they require a new column.

**Comment by michelson** [comment]
@georgeclaghorn I've change the release notes, but added the rails update command in the notes, if you want I can remove that part.

question: what will happen to the files that are already uploaded, I mean before the db migrations to multiservice column ? in production application should we take some precautions ?

**Comment by DmitryTsepelev** [comment]
> question: what will happen to the files that are already uploaded, I mean before the db migrations to multiservice column ? in production application should we take some precautions ?

The generated [migration](https://github.com/rails/rails/pull/34935/files#diff-07ef0a019eb0be7f78b64041f49d5821147e6fec2c6b54dacb68d37ac198f04aR7) will update all existing blobs with the current configured service name. However, I've realized that updating a lot of data (which might be a case for the `blobs` table) as well as adding a `null: false` constraint might take a lot of time, should we add a note suggesting to review the generated migration and change it if needed? (e.g., updating images in batches instead of `update_all`)

**Comment by michelson** [comment]
@dmitry I agree, I have an application already updated with a lot of images. I will check how long it takes and add some notes here if maintainers are agree

**Comment by michelson** [comment]
well I thought that I had more blobs that I actually have. I'd run the migrations and the process was smooth for ~1000 records. 
```
== 20201210184517 AddServiceNameToActiveStorageBlobs: migrated (0.0610s) ======
== 20201210184518 CreateActiveStorageVariantRecords: migrated (0.0259s) =======
```

**Comment by DmitryTsepelev** [comment]
Well 1000 images is not a big deal, I feel like 100k could be a problem

File: guides/source/6_1_release_notes.md [hunk 1]
```diff
@@ -308,6 +308,7 @@ Please refer to the [Changelog][active-storage] for detailed changes.
 *   Add `Blob.create_and_upload` to create a new blob and upload the given `io`
     to the service.
     ([Pull Request](https://github.com/rails/rails/pull/34827))
+*   ActiveStorage::Blob#service_name column was added, ( `bin/rails app:update` required ).
```
Comment (rafaelfranca) [inline guides/source/6_1_release_notes.md:311]:
```suggestion
*   `ActiveStorage::Blob#service_name` column was added. It is required that a migration is run after the upgrade. Run `bin/rails app:update` to generate that migration.
```

**Review by rafaelfranca — CHANGES_REQUESTED** [review]

**Comment by reachlin** [comment]
we have more than 100k images, what's the recommended steps here?

**Pull request merged**